README.md

```diff
@@ -125,19 +130,90 @@ All used config files, logs and results are stored in a folder in the ```benchma
 #### Non-Interactive Mode
 It is also possible to start the script with a config file instead of using the interactive mode:
 ```
- python src/main.py --config test_config.yml
+ python src/main.py --config config.yml
 ```
 
 > __Note:__ This should only be used by experienced users as invalid values will cause the framework to fail!
 
+Example for a config file:
+
+```
+application:
+  config:
+    nodes:
+    - 3
+  name: TSP
+  submodules:
+  - config: {}
+    name: GreedyClassicalTSP
+    submodules:
+    - config: {}
+      name: Local
+      submodules: []
+repetitions: 1
+```
+
 
 #### Summarizing Multiple Existing Experiments
 You can also summarize multiple existing experiments like this:
 ```
-python src/main.py --summarize /Users/user1/quark/benchmark_runs/2021-09-21-15-03-53 /Users/user1/quark/benchmark_runs/2021-09-21-15-23-01
+python src/main.py --summarize quark/benchmark_runs/2021-09-21-15-03-53 quark/benchmark_runs/2021-09-21-15-23-01
 ```
 This allows you to generate plots from multiple experiments.
 
+## Dynamic Imports
```

Comment:
This part is supposed to be only in readthedocs as it is not considered necessary to get started with QUARK. There is a hint to that in line 63 in this readme which we think is enough.